src/main/kotlin/org/rust/lang/core/psi/util/RustAttrExtensions.kt

```diff
@@ -0,0 +1,65 @@
+package org.rust.lang.core.psi.util
+
+import org.rust.lang.core.psi.*
+
+/**
+ * Appends a new meta item `new` to an attribute with a list of arguments:
+ *
+ * ```
+ * #[attr]              ->  #[attr(new)]
+ * #[attr="something"]  ->  #[attr(new)]
+ * #[attr(something)]   ->  #[attr(something, new)]
+ * #[attr(something]    ->  #[attr(something, new]
+ * #[attr()]            ->  #[attr(new)]
+ * #[attr(]             ->  #[attr(new]
+ * ```
+ *
+ * @return `meta` argument or its copy if insertion was successful, null otherwise.
+ */
+fun RustAttr.appendMetaToList(meta: RustMetaItem): RustMetaItem? {
+    // extract attribute internals
+    val attrMeta = when (this) {
```

Comment:
Fixed in https://github.com/intellij-rust/intellij-rust/commit/1c609f91f37fbd0d8ab39c1066ddd3c7b55145a6